OrbitGl/GlCanvas.h

```diff
@@ -50,6 +50,7 @@ class GlCanvas {
   Vec2 WorldToScreen(Vec2 world_pos) const;
   int WorldToScreenHeight(float height) const;
   int WorldToScreenWidth(float width) const;
+  Vec2 ScreenQtToScreenGl(Vec2 qt_pos) const;
```

Comment:
[[nodiscard]] is missing - but it's also missing for all methods above, we could solve this in a seperate cleanup PR.

Reply:
Perfect, I will make a new tiny-PR to merge it in master!